classes/submission/reviewAssignment/ReviewAssignment.inc.php

```diff
@@ -469,16 +469,25 @@ function getStatus() {
 		if ($this->getDeclined()) {
 			return REVIEW_ASSIGNMENT_STATUS_DECLINED;
 		} elseif (!$this->getDateCompleted()) {
+			$dueTimes = array_map(function($dateTime) {
+					// If no due time, set it to the end of the day
+					if (substr($dateTime, 11) === '00:00:00') {
+						$dateTime = substr($dateTime, 0, 11) . '23:59:59';
+					}
+					return strtotime($dateTime);
+				}, array($this->getDateResponseDue(), $this->getDateResponseDue()));
```

Comment:
should the second value be getDateDue() instead of the same getDateResponseDue() ?